### DISCUSSION TITLE
Are additional downscaling methods required to interpret high resolution ensemble forecasts?

### DISCUSSION THREAD

**Discussion by guidocioni** [open-meteo/open-meteo#955] [Q&A]
It's not the first time I notice that `icon-d2-eps` produces way less precipitation than its parent nests `icon-eu-eps` and `icon-eps`. For example, here is the expected arrival of thunderstorms tomorrow over Hamburg

![newplot (2)](https://github.com/user-attachments/assets/ec20e887-df3b-4d4d-b006-d07683ca8910)
![newplot (3)](https://github.com/user-attachments/assets/78c7344f-14b4-4a69-8ef3-87f3c1e63790)
![newplot (4)](https://github.com/user-attachments/assets/98ad4b2e-cc7d-4d14-99ed-7e654c3680d4)

As you can see, many more members have precipitation in the low resolution nests, while `icon-d2` only has some. This is **not** negligible as it impacts precipitation probability (computed as `number of members having hourly precip > threshold / total number of members`) which varies from 5-15% in `icon-d2-eps` to 25-60% in `icon-eu-eps`. This is a significant difference that would lead to a completely different interpretation of the forecast.

I've been trying to explain to myself this discrepancy and how to mitigate this. I don't think it is a problem in the raw data ingestion, because the difference in the model can be seen also in maps

![prob_snow_30](https://github.com/user-attachments/assets/b558459f-15e0-4513-ac5c-06a58e98cc15)
![prob_snow_39](https://github.com/user-attachments/assets/0dc5af3c-cc61-4ef3-8227-2300e0feb0fb)

I think the reason is instead mainly related to the fact that `icon-d2-eps` explicitly resolves convection and creates a completely different pattern of precipitation w.r.t to `icon-eu-eps` (and all low resolution models in general). For this reason, precipitating cells in `icon-d2-eps` are more local and it could be that, by just querying a specific grid point, one is missing the thunderstorm that is just passing by a few kilometers shift. However, the differences could also just be related to the fact that the models predict completely different outcomes.

That's why I've been wondering whether is really beneficial to use only one grid point when employing high resolution ensemble models, or whether this data necessarily need to be always statistically downscaled to provide a good interpretation. For example, in the same way `icon-eu-eps` artificially "smooths" precipitation over a large area, one may need to request multiple grid points from `icon-d2-eps` to get a similar answer which is more relevant than just querying a single point.

What do you think @patrick-zippenfenig ? Anyone ever thought about this problem and a possible solution?

**Comment by phillipseamore** [comment]
Without knowing what the intended use case is I'd think an average of average from all three should get you close to solving spatial and temporal variance without requiring neighborhood processing, you could use different weights for each or cap them (like global can never contribute more than 50% chance). You could also include the previous and next step with different weights, remember that precip is accumulated for the previous hour (0.25mm at hour 15 is precip modeled in the time from hour 14 to 15).

For end user products this can get complicated, see ECCC's WEonG (from deterministic models) https://collaboration.cmc.ec.gc.ca/cmc/cmoi/product_guide/docs/tech_notes/technote_weong-hrdps_e.pdf or some of the ensemble post-processing tools.
